tck/models/account-key.ts

```diff
@@ -0,0 +1,11 @@
+export interface AccountKeyGenerationParams {
+    readonly type: string;
+    readonly fromKey?: string;
+    readonly threshold?: number;
+    readonly keys?: AccountKeyGenerationParams[];
+}
+
+export interface AccountKeyGenerationResponse {
```

Comment:
The reason I organized the models this way is primarily to avoid certain TypeScript anti-patterns, like over-segregating types, which can sometimes lead to unnecessary complexity and harder maintainability. However, I completely agree with you that for the sake of consistency and to align with the existing TCK projects structure, it's important to maintain a unified approach. I’ll go ahead and refactor the code accordingly and update the PR.